bazel_codegen/pubspec.yaml

```diff
@@ -5,14 +5,17 @@ homepage: https://github.com/dart-lang/bazel
 version: 0.1.6
 
 environment:
-  sdk: ">=1.8.0 <2.0.0"
+  sdk: ">=1.25.0-dev.7.0 <2.0.0"
```

Comment:
hmm, we'll probably end up wanting to cut a release that works with build v10 on a non-dev SDK... I'll file an issue.

What are we relying on here?